cycler.py

```diff
@@ -389,6 +389,55 @@ def simplify(self):
         trans = self._transpose()
         return reduce(add, (_cycler(k, v) for k, v in six.iteritems(trans)))
 
+    def concat(self, other):
+        """Concatenate two cyclers.
```

Comment:
Great docstring - I'd be inclined to include the simplest possible example though, so that people can get a visual grasp of what is going on.